pkg/executor/point_get_test.go

```diff
@@ -375,40 +375,3 @@ func TestWithTiDBSnapshot(t *testing.T) {
 
 	tk.MustQuery("select * from xx").Check(testkit.Rows("1", "7"))
 }
-
-func TestGlobalIndexPointGet(t *testing.T) {
```

Comment:
Move all global index related integration test to `tests/integrationtest/t/globalindex`